src/module/item/entity.js

```diff
@@ -82,12 +82,12 @@ export class OseItem extends Item {
   rollWeapon(options = {}) {
     let isNPC = this.actor.data.type != "character";
```

Comment:
`this.actor.type` ? Or is this `this.parent.type`?

Reply:
not sure when this became outdated, I think this was an early one fixed in one of last month's commits